modulesets-stable/gtk-osx-gstreamer.modules

```diff
@@ -24,8 +24,8 @@
 
   <autotools id="gstreamer" autogenargs="--disable-tests" supports-non-srcdir-builds="no"
     makeargs="ERROR_CFLAGS=" autogen-sh="configure">
-    <branch module="gstreamer/1.10/gstreamer-1.10.4.tar.xz" version="1.10.4"
-            hash="sha256:50c2f5af50a6cc6c0a3f3ed43bdd8b5e2bff00bacfb766d4be139ec06d8b5218">
+    <branch module="gstreamer/gstreamer-1.10.5.tar.xz" version="1.10.5"
```

Comment:
ftp.gnome.org module paths need the '1.10' indicating the major/minor version. gstreamer.freedesktop.org paths don't. In this moduleset ftp.gnome.org is the default, so you need to either restore the 1.10 or add ```repo="gstreamer"``` to the ```branch``` element for gstreamer, gst-plugins-base, and gst-plugins-good.

Reply:
Sorry about that, I'll edit it, I was wondering why your URLs look different. Now I know!

Reply:
Done: